tests/test_models.py

```diff
@@ -3177,7 +3177,7 @@ def test_replace_contributor(self):
     def test_permission_override_on_readded_contributor(self):
 
         # A child node created
-        self.child_node = NodeFactory(parent=self.project, creator=self.auth)
+        self.child_node = NodeFactory(parent=self.project, creator=self.auth.user)
```

Comment:
So, it's fixed here because `node.creator` is now accessed upon node creation, and this test was previously setting up `creator` incorrectly.